tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -316,6 +321,22 @@ describe(__filename, () => {
     expect(link).toHaveProp('to', '/addon/chill-out/privacy/');
   });
 
+  it('adds the `src` query parameter to the privacy policy link if available in the location', () => {
+    const src = 'some-src';
+
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      has_privacy_policy: true,
+    });
+    const root = render({
+      addon,
+      location: createFakeLocation({ query: { src } }),
+    });
+    const link = root.find('.AddonMoreInfo-privacy-policy').find(Link);
+
+    expect(link).toHaveProp('to', `/addon/chill-out/privacy/?src=${src}`);
+  });
+
```

Comment:
same question re: testing the missing/empty `src` cases with the new tests in this file.